App.js

```diff
@@ -0,0 +1,23 @@
+import TopBar from './modules/TopBar.js';
+import NavBottom from './modules/NavBottom.js';
+import ProfileInfo from './modules/ProfileInfo.js';
+import UserButtons from './modules/UserButtons.js';
+import UserStoryInfo from './modules/UserStoryInfo.js';
+import Gallery from './modules/Gallery.js';
+
+
+const App = () => 
+        {
```

Comment:
I don't understand the indentation used here. I'm counting 7 spaces.
I'll recommend 2 instead.

Reply:
ok  I will fix them